src/celeritas/ext/detail/ImportProcessConverter.cc

```diff
@@ -137,7 +138,7 @@ ImportModelClass to_import_model(const G4VEmModel& model)
         {"eBremLPM",            ImportModelClass::e_brems_lpm},
         {"eplus2gg",            ImportModelClass::e_plus_to_gg},
         {"LivermorePhElectric", ImportModelClass::livermore_photoelectric},
-        {"KleinNishina",        ImportModelClass::klein_nishina},
+        {"Klein-Nishina",       ImportModelClass::klein_nishina},
```

Comment:
Geant4 is the worst...

Reply:
Clearly the "-" stands for "minus atomic shell effects"